Dockerfile.osx

```diff
@@ -8,37 +8,20 @@ RUN if [ -z "${mono_version}" ]; then echo -e "\n\nargument mono-version is mand
     dnf clean all && \
     git clone https://github.com/tpoechtrager/osxcross.git && \
     cd /root/osxcross && \
+    git checkout 542acc2ef6c21aeb3f109c03748b1015a71fed63 && \
     ln -s /root/files/MacOSX10.14.sdk.tar.xz /root/osxcross/tarballs && \
-    UNATTENDED=1 ./build.sh ;\
-    cd /root && \
-    git clone https://github.com/tpoechtrager/apple-libtapi.git && \
-    cd apple-libtapi && \
-    INSTALLPREFIX=/root/osxcross/target ./build.sh && ./install.sh && \
-    cd /root && \
-    git clone https://github.com/tpoechtrager/cctools-port.git && \
-    cd cctools-port/cctools/ && \
-    ./configure --prefix=/root/osxcross/target --target=x86_64-apple-darwin17 --with-libtapi=/root/osxcross/target && \
-    make -j && make install && \
-    cd /root && \
-    ln -fs /root/osxcross/target/bin/x86_64-apple-darwin17-install_name_tool /root/osxcross/target/bin/install_name_tool && \
-    ln -fs /root/osxcross/target/bin/x86_64-apple-darwin17-ar /root/osxcross/target/bin/ar && \
-    cd /root/osxcross/ && \
-    ./build_compiler_rt.sh && \
-    export CLANG_LIB_DIR=$(clang -print-search-dirs | grep "libraries: =" | tr '=' ' ' | tr ':' ' ' | awk '{print $2}') && \
-    mkdir -p ${CLANG_LIB_DIR}/include && \
-    mkdir -p ${CLANG_LIB_DIR}/lib/darwin && \
-    cp -rv /root/osxcross/build/compiler-rt/include/sanitizer ${CLANG_LIB_DIR}/include && \
-    cp -v /root/osxcross/build/compiler-rt/build/lib/darwin/*.a ${CLANG_LIB_DIR}/lib/darwin && \
-    cp -v /root/osxcross/build/compiler-rt/build/lib/darwin/*.dylib ${CLANG_LIB_DIR}/lib/darwin
+    UNATTENDED=1 ./build.sh
```

Comment:
Fix the stay `;` instead of `&&`.

I noticed that we don't clean things up after the installation, so we still have the `osxcross` git repo and build objects I assume. I guess we could only keep `/root/osxcross/target/` and delete the rest?